enterprise/server/test/integration/workflow/workflow_test.go

```diff
@@ -61,6 +61,8 @@ func setup(t *testing.T, gp interfaces.GitProvider) (*rbetest.Env, interfaces.Wo
 			env.SetInvocationSearchService(iss)
 		},
 	})
+	// Configure executors so that we can run at least one workflow task.
+	flags.Set(t, "executor.memory_bytes", fmt.Sprintf("%d", 10_000_000_000))
```

Comment:
please link to / document where this 10G number comes from?

// Workflow tasks require 8Gi to schedule, but all of our workflow test cases
// typically use far less RAM. To enable this test to run on machines smaller
// than 8Gi, force-set the executor memory_bytes to 10Gi so the workflow
// tasks will schedule.

Reply:
Thanks! Done